clippy_tests/examples/doc.stderr

```diff
@@ -206,6 +206,14 @@ error: you should put `be_sure_we_got_to_the_end_of_it` between ticks in the doc
     |
     = note: `-D doc-markdown` implied by `-D warnings`
 
+error: you should put `http://www.unicode.org/reports/tr9/#Reordering_Resolved_Levels` between ticks in the documentation
```

Comment:
I think we should add some simple heuristic to suggest surrounding it with `<>`. Something like "anything that begins with `{ascii}*://` or `www.`"